### PR TITLE
[docs] update tutorial for Theta sketches

### PR DESCRIPTION
**Pull request by edgar2020**
### Description

Updating `tutorial-sketches-theta.md` to to use batch-sql ingestion. 

This PR has:

- [X] been self-reviewed.

### CODE REVIEW

File: docs/tutorials/tutorial-sketches-theta.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Imagine you are interested in the number of visitors that watched episodes of a
 
 There is no way to answer these questions by just looking at the aggregated numbers. You would have to go back to the detail data and scan every single row. If the data volume is high enough, this may take a very long time, meaning that an interactive data exploration is not possible.
 
-An additional nuisance is that unique counts don't work well with rollups. For this example, it would be great if you could have just one row of data per 15 minute interval[^1], show, and episode. After all, you are not interested in the individual user IDs, just the unique counts.
+An additional nuisance is that unique counts don't work well with rollups. For this example, it would be great if you could have just one row of data per 15 minute interval<super>[^1]</super>, show, and episode. After all, you are not interested in the individual user IDs, just the unique counts.
```
Comment (asdf2014) [inline docs/tutorials/tutorial-sketches-theta.md:39]:
```suggestion
An additional nuisance is that unique counts don't work well with rollups. For this example, it would be great if you could have just one row of data per 15 minute interval<sup>[^1]</sup>, show, and episode. After all, you are not interested in the individual user IDs, just the unique counts.
```

**Review by asdf2014 — COMMENTED** [review]
`<super>` is invalid as an HTML tag, the correct one should be `<sup>`

File: docs/tutorials/tutorial-sketches-theta.md [hunk 2]
```diff
@@ -209,36 +137,23 @@ Let's first see what the data looks like in Druid. Run the following SQL stateme
...
-       THETA_SKETCH_ESTIMATE(DS_THETA(theta_uid)) AS users
-FROM   ts_tutorial
-GROUP  BY 1, 2, 3
+SELECT
+  __time,
+  "show",
+  "episode",
+  THETA_SKETCH_ESTIMATE(theta_uid) AS users
+FROM ts_tutorial
+GROUP BY 1, 2, 3, 4
```
Comment (vtlim) [inline docs/tutorials/tutorial-sketches-theta.md:153]: Double check SQL standard on this. I don't think you need the GROUP BY since you're just selecting and applying a scalar function. When I try the query without GROUP BY, I get the same thing.

File: docs/tutorials/tutorial-sketches-theta.md [hunk 3]
```diff
@@ -256,7 +181,10 @@ SELECT THETA_SKETCH_ESTIMATE(
 FROM ts_tutorial
 ```
 
-![Count distinct with Theta sketches and filters](../assets/tutorial-theta-08.png)
+The `APPROX_COUNT_DISTINCT_DS_THETA` function applies the following:
+
+* `DS_THETA`: Creates a new Theta sketch from the column of Theta sketches.
+* `THETA_SKETCH_ESTIMATE`: Calculates the distinct count estimate from the output of `DS_THETA` where the show is _Bridgerton_.
```
Comment (vtlim) [inline docs/tutorials/tutorial-sketches-theta.md:187]: >where the show is _Bridgerton_.

Shouldn't be in this description. The FILTER part doesn't belong to THETA_SKETCH_ESTIMATE

File: docs/tutorials/tutorial-sketches-theta.md [hunk 4]
```diff
@@ -274,7 +202,7 @@ SELECT THETA_SKETCH_ESTIMATE(
 FROM ts_tutorial
 ```
 
-![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-09.png)
```
Comment (vtlim) [inline docs/tutorials/tutorial-sketches-theta.md:277]: Why did you delete these three images?

image

File: docs/tutorials/tutorial-sketches-theta.md (hunk 2 above)
Comment (edgar2020) [reply docs/tutorials/tutorial-sketches-theta.md:157]: You are correct. It appears that the GROUP BY is unnecessary because it runs just fine and gives the same output compared to if it did have it

File: docs/tutorials/tutorial-sketches-theta.md (hunk 4 above)
Comment (edgar2020) [reply docs/tutorials/tutorial-sketches-theta.md:277]: Resolved

File: docs/tutorials/tutorial-sketches-theta.md [hunk 5]
```diff
@@ -95,103 +97,37 @@ date,uid,show,episode
...
+FROM "source"
+GROUP BY 1, 2, 3
+PARTITIONED BY DAY
 ```
 
-Notice the `theta_uid` object in the `metricsSpec` list, that defines the `thetaSketch` aggregator on the `uid` column during ingestion.
+Notice that there is no `uid` in the `SELECT` statement.
+
+In this scenario you are not interested in individual user IDs, only the unique counts.
+
```
Comment (vtlim) [inline docs/tutorials/tutorial-sketches-theta.md:125]:
```suggestion
In this scenario you are not interested in individual user IDs, only the unique counts.
```

File: docs/tutorials/tutorial-sketches-theta.md [hunk 6]
```diff
@@ -95,103 +97,37 @@ date,uid,show,episode
...
+Instead you create Theta sketches on the values of `uid` using the `DS_THETA` function.
```
Comment (vtlim) [inline docs/tutorials/tutorial-sketches-theta.md:126]:
```suggestion
In this scenario you are not interested in individual user IDs, only the unique counts.
Instead you create Theta sketches on the values of `uid` using the `DS_THETA` function.
```

File: docs/tutorials/tutorial-sketches-theta.md [hunk 7]
```diff
@@ -209,36 +145,22 @@ Let's first see what the data looks like in Druid. Run the following SQL stateme
 SELECT * FROM ts_tutorial
 ```
 
-![View data with SELECT all query](../assets/tutorial-theta-06.png)
+![View data with SELECT all query](../assets/tutorial-theta-03.png)
```
Comment (techdocsmith) [inline docs/tutorials/tutorial-sketches-theta.md:148]: nit/not necessary for this pr: In general, I think tables represent query results better than screen shots. And you can also wrap the results in a `view results` expander/details section to make the overall document smaller.

**Review by techdocsmith — CHANGES_REQUESTED** [review]
Good refactor for MSQ. Be careful when removing content that might be useful (line 122). That's the only fix required for this pr.

**Review by vtlim — APPROVED** [review]

**Review by asdf2014 — APPROVED** [review]
